assets/_sass/_structure.scss

```diff
@@ -65,305 +65,303 @@
+      .presidium-suggested-reading,
+      .presidium-suggested-courses,
+      .presidium-best-practice,
+      .presidium-warning,
+      .presidium-chris-says {
+        padding: 9px 10px 10px;
+        margin-bottom: 10px;
+        border: 2px solid;
+        border-radius: 4px;
+        font-style: normal;
+
+        em {
+          font-weight: bold;
+          font-style: normal;
         }
 
-        .presidium-pro-tip {
-          @include callout($color-orange, 'span pro tip');
+        a {
+          text-decoration: underline;
+          color: $color-grey;
         }
 
-        .presidium-for-example {
-          @include callout($color-green, 'for example...');
+        span {
+          font-size: 18px;
+          line-height: 21px;
+          font-weight: 500;
+          letter-spacing: 0;
+          margin-bottom: 4px;
+          display: flex;
         }
 
-        .presidium-suggested-reading {
-          @include callout($color-blue, 'suggested reading');
+        &::before {
+          display: flex;
+          font-size: 13px;
+          font-weight: 500;
+          letter-spacing: 0;
+          line-height: 15px;
+          text-transform: uppercase;
+          margin-bottom: 4px;
         }
+      }
 
-        .presidium-suggested-courses {
-          @include callout($color-blue, 'suggested courses');
-        }
+      .presidium-pro-tip {
+        @include callout($color-orange, 'span pro tip');
+      }
 
-        .presidium-best-practice {
-          @include callout($color-orange, 'best practice');
-        }
+      .presidium-for-example {
+        @include callout($color-green, 'for example...');
+      }
 
-        .presidium-warning {
-          @include callout($color-red, 'warning');
-        }
+      .presidium-suggested-reading {
+        @include callout($color-blue, 'suggested reading');
+      }
 
-        .presidium-prerequisite {
-          @include callout($color-yellow-green, 'prerequisite');
-        }
+      .presidium-suggested-courses {
+        @include callout($color-blue, 'suggested courses');
+      }
 
-        .presidium-chris-says {
-          background-color: #000000;
-          border-color: #000000;
-          background-image: url('images/pentagram-symbol.svg');
-          color: $color-grey-2;
-          background-repeat: no-repeat;
-          background-position: top right;
-          background-size: 31px;
-          background-origin: content-box;
+      .presidium-best-practice {
+        @include callout($color-orange, 'best practice');
+      }
 
-          @include callout($color-grey-2, 'chris says...');
-        }
+      .presidium-warning {
+        @include callout($color-red, 'warning');
+      }
+
+      .presidium-prerequisite {
+        @include callout($color-yellow-green, 'prerequisite');
+      }
+
+      .presidium-chris-says {
+        background-color: #000000;
+        border-color: #000000;
+        background-image: url('images/pentagram-symbol.svg');
```

Comment:
So we're using Presidium to summon Satan now?

Reply:
Lol, this explains a lot